### ISSUE TITLE
feat(hapi_lut): factorised gas × surface × scattering LUT and two-way path test

### ISSUE THREAD

**Issue by jejjohnson** [jejjohnson/plumax#99]
## Problem / Request
`hapi_lut` builds per-gas cross-section LUTs over (T, P) and `multi.py` combines gases; `radtran/nb_lut.py` tabulates normalised brightness. The design's factorised LUT (`hapi_lut.factorised`) separates gas absorption, surface and scattering so each axis stays small and the product is the training-data generator for the neural RTM. Separately, the design table flags `beers.py` "✓ — add two-way path if not already": `air_mass_factor` exists but there is no test pinning nadir `τ_total = 2τ` and oblique AMF = 1/cos SZA + 1/cos VZA.

## User Story
> As a user building a scene-class emulator, I want a LUT factorised as `T_gas(ν; T, P, column) · S(λ; albedo, geometry) · Φ(λ; AOD)` with out-of-box states failing loudly, so the training set covers the corners without a full-tensor blow-up.

## Proposed API
```python
# src/plumax/hapi_lut/factorised.py
@dataclass(frozen=True)
class FactorisedLUT:
    gas: xr.Dataset          # σ(ν; T, P) per gas — existing generator output
    surface: xr.Dataset      # per-band albedo basis
    scattering: xr.Dataset   # AOD → per-band transmittance factor (v1: clear-sky table only)
    def sample(self, state: dict, *, strict: bool = True) -> jax.Array: ...   # raises outside the bounding box when strict
def build_factorised_lut(config: LUTConfig, *, surface_grid, aod_grid) -> FactorisedLUT: ...
```

## Implementation Steps
- [ ] Add `hapi_lut/factorised.py` reusing `generator.py` for the gas axis and the coverage validation from #44
- [ ] Bounding-box check with a clear error (design: OOD states "fail loudly, not silently extrapolate")
- [ ] Add the two-way path consistency test against `beers.air_mass_factor` / `transmittance`

## Definition of Done
- [ ] LUT vs HAPI in-distribution radiance interpolation error < 1 %
- [ ] Out-of-box `sample(strict=True)` raises; `strict=False` clamps and warns
- [ ] Two-way path test passes
- [ ] `make test && make lint && make typecheck` green

## Testing
- [ ] `tests/hapi_lut/test_factorised.py::test_in_distribution_under_1pct`
- [ ] `tests/hapi_lut/test_factorised.py::test_ood_raises`
- [ ] `tests/hapi_lut/test_beers.py::test_two_way_path_amf`

## Relationships
- Parent (theme epic): #73
- Blocks: neural RTM (training-data generator)
- Related: #44, #45 (LUT coverage / cache, closed)
